apps/api/src/app/workflows-v2/workflow.controller.ts

```diff
@@ -161,6 +164,23 @@ export class WorkflowController {
     );
   }
 
+  @Get('/:workflowId/step/:stepUuid/metadata')
```

Comment:
Small alignment with the naming.

```suggestion
  @Get('/:workflowId/steps/:stepId/metadata')
```